### ISSUE TITLE
Maven Goal `citrus:create-test` generate test class with compile error

### ISSUE THREAD

**Issue by sparsick** [citrusframework/citrus#505]
**Citrus Version**
2.7.7 with Java 10 and JUnit 5

**Expected behavior**
Test class without compile error

**Actual behavior**
Generate following code:
```
package com.consol.citrus;

import com.consol.citrus.annotations.CitrusTest;
import org.junit.jupiter.api.Test;
import org.junit.jupiter.api.extension.ExtendWith;

/**
 *
 *
 * @author Sandra Parsick
 * @since 2018-10-04
 */
@ExtendWith(com.consol.citrus.junit.jupiter.CitrusExtension)
public class MailIT {
	@CitrusTest
	@Test
	public void mail() {
		echo("TODO: Code the test Mail");

	}
}

```

**Test case sample**
1) Run `mvn citrus:create-test`
2) Add following input 
```
Enter test name:: FooIT
Enter test author: Unknown: : 
Enter test description: : : 
Enter test package: com.consol.citrus: : 
Choose unit test framework: (testng/junit4/junit5) testng: : junit5
Choose target code base type: (java/xml) java: : java
Create test with XML schema? (y/n) n: : n
Create test with WSDL? (y/n) n: : n
Create test with Swagger API? (y/n) n: : n
```

**Comment by svettwer** [comment]
Hi!

Thx for the detailed bug report! :+1: 

It was possible for me to reconstruct the issue.
```java
/**
 * foo
 *
 * @author bar
 * @since 2018-10-07
 */
@ExtendWith(com.consol.citrus.junit.jupiter.CitrusExtension)
public class FooBar {
    @CitrusTest
    @Test
    public void foobar() {
        echo("TODO: Code the test FooBar");

    }
}
```

The generated test is not executable because of multiple issues.
1. `com.consol.citrus.junit.jupiter.CitrusExtension` should be `com.consol.citrus.dsl.junit.jupiter.CitrusExtension.class`
2. `public void foobar()` should receive a designer or a runner to be able to execute the `echo` command of the test.

A running test would be:
```java
@ExtendWith(com.consol.citrus.dsl.junit.jupiter.CitrusExtension.class)
public class FooBar {

    @CitrusTest
    @Test
    public void foobar(@CitrusResource TestRunner testRunner) {
        testRunner.echo("TODO: Code the test FooBar");

    }
}
```

BR,
Sven

**Comment by svettwer** [comment]
Fixed by merging #512 

I'll open a enhancement story as a follow up to allow the user to choose between the `TestDesigner` and `TestRunner`.

BR,
Sven
